projects/plugins/jetpack/modules/subscriptions/subscribe-modal/class-jetpack-subscribe-modal.php

```diff
@@ -244,9 +224,9 @@ public function has_subscription_cookie() {
  *
  * @since 12.4
  *
- * @param bool Defaults to false.
+ * @param bool Defaults to true.
  */
-if ( apply_filters( 'jetpack_subscriptions_modal_enabled', false ) ) {
+if ( apply_filters( 'jetpack_subscriptions_modal_enabled', true ) ) {
```

Comment:
Should we deprecate that filter in this PR, in a follow-up PR, or would you rather we keep it around for a bit longer?

Reply:
Any objections deprecating already in this PR?

What's the Jetpack way of deprecating a filter?

Reply:
I think we could remove the filter in this PR. 👍

Since it's unlikely that hook has been really used in the wild outside of Jetpack, I would just remove it, and add it to the list here just in case:
https://github.com/Automattic/jetpack/blob/d19049e08efdf0551d02825d552399b9d3b284d2/projects/plugins/jetpack/class.jetpack.php#L5848-L5851

Reply:
Moved the filter to deprecated hooks 👍  Thanks for the tip!